### PR TITLE
OGCS: Change to beta version

### PR DESCRIPTION
**Pull request by linsui**

### CODE REVIEW

File: bucket/outlook-google-calendar-sync.json [hunk 1]
```diff
@@ -14,11 +13,10 @@
     ],
     "persist": "settings.xml",
     "checkver": {
-        "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/",
```
Comment (Ash258) [inline bucket/outlook-google-calendar-sync.json]: Keep releases

**Comment by linsui** [comment]
I found it seems nupkg will be released later than zip. Yesterday there is only zip and exe. And in the past only beta version has nupkg. Don't know will there be a nupkg in the next release.

File: bucket/outlook-google-calendar-sync.json [hunk 2]
```diff
@@ -2,10 +2,9 @@
     "version": "2.8.1-alpha",
     "description": "Sync your Outlook and Google calendars.",
     "license": "MPL-2.0",
-    "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/download/v2.8.1-alpha/OutlookGoogleCalendarSync-2.8.1-alpha-full.nupkg#/dl.7z",
+    "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/download/v2.8.1-alpha/Portable_OGCS_v2.8.1.zip",
     "homepage": "https://phw198.github.io/OutlookGoogleCalendarSync/",
```
Comment (Ash258) [inline bucket/outlook-google-calendar-sync.json]: Move homepage above URL

File: bucket/outlook-google-calendar-sync.json [hunk 3]
```diff
@@ -14,11 +13,10 @@
     ],
     "persist": "settings.xml",
     "checkver": {
-        "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/",
-        "re": "/releases/download/(?:v)(?<version>.*?)/"
+        "url": "https://github.com/phw198/OutlookGoogleCalendarSync/tags",
+        "regex": "tag/v([\\w.-]+)"
```
Comment (Ash258) [inline bucket/outlook-google-calendar-sync.json]:
```suggestion
        "regex": "tag/v([\\w.-]+)"
```

File: bucket/outlook-google-calendar-sync.json [hunk 4]
```diff
@@ -14,11 +13,10 @@
     ],
     "persist": "settings.xml",
     "checkver": {
-        "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/",
-        "re": "/releases/download/(?:v)(?<version>.*?)/"
+        "url": "https://github.com/phw198/OutlookGoogleCalendarSync/tags",
```
Comment (Ash258) [inline bucket/outlook-google-calendar-sync.json]:
```suggestion
        "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases",
```

File: bucket/outlook-google-calendar-sync.json [hunk 5]
```diff
@@ -2,10 +2,9 @@
     "version": "2.8.1-alpha",
     "description": "Sync your Outlook and Google calendars.",
     "license": "MPL-2.0",
-    "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/download/v2.8.1-alpha/OutlookGoogleCalendarSync-2.8.1-alpha-full.nupkg#/dl.7z",
+    "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/download/v2.8.1-alpha/Portable_OGCS_v2.8.1.zip",
```
Comment (Ash258) [inline bucket/outlook-google-calendar-sync.json]: Do not use tags. Here you can see that this file does not exists. Tags does not explicitly mean there is release available for this tag.

A

**Pull request merged**